src/styles/variables.css

```diff
@@ -69,6 +69,10 @@
    * Confirm deletion bg
    */
   --color-confirm: #E24A4A;
+  --color-confirm-hover: #CE4343;
+
+  --color-line-active: #D8E8FA;
+  --color-line-gray: #EFF0F1;
```

Comment:
so please derpecate the `--color-gray-border`;

Reply:
no answer

Reply:
added deprecated comment